### PR TITLE
Added Engity's Bifröst

### PR DESCRIPTION
**Pull request by blaubaer**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [ ] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/bifroest.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+platforms:
+  - Go
+  - Docker
+tags:
+  - Remote Access
+  - Proxy
+source_code_url: https://github.com/engity-com/bifroest
+stargazers_count: 12
+updated_at: '2024-11-04'
+archived: false
```
Comment (Ki-er) [inline software/bifroest.yml:15]: Added automagically so can be removed for now.
```suggestion
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Quick change.

File: software/bifroest.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+website_url: https://bifroest.engity.org/
+description: Highly customizable SSH server with several ways to authorize a user and options where and how to execute a user's session.
+licenses:
+  - Apache-2.0
+platforms:
+  - Go
+  - Docker
+tags:
+  - Remote Access
+  - Proxy
```
Comment (nodiscc) [inline software/bifroest.yml:11]:
```suggestion
tags:
  - Remote Access
```

Not relevant, see tag description

> A [proxy](https://en.wikipedia.org/wiki/Proxy_server) is a server application that acts as an intermediary between a client requesting a resource and the server providing that resource. This section about forward (i.e. outgoing) proxies.

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/bifroest.yml (hunk 2 above)
Comment (nodiscc) [reply software/bifroest.yml:11]: > This section about forward (i.e. outgoing) proxies.


My comment still stands. OpenSSH also has forwarding/proxy-like capabilities but it still doesn't count as a proxy in the context of this list.

Comment (blaubaer) [reply software/bifroest.yml]: @nodiscc updated as requested.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**